components/table/test/table-test-sticky-visual-diff.js

```diff
@@ -0,0 +1,368 @@
+import '../../../components/inputs/input-number.js';
+import '../../../components/inputs/input-text.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { tableStyles } from '../table-wrapper.js';
+
+const url = new URL(window.location.href);
+const type = url.searchParams.get('type') === 'light' ? 'light' : 'default';
+
```

Comment:
All the visual-diff stuff had to move into Lit components so that they could apply the table styles. The tests themselves are the same.